### DISCUSSION TITLE
Como se escreve Front-end?

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#934] [Discussão]
Front-end?
Front End?
Front-End?

Qual seria a forma correta?
Cada vaga que vejo escreve de uma forma, nos sites a mesma coisa.

**Comment by luizbills** [comment]
Se alguém descobriu, ele não contou para ninguém.

**Comment by rubenmarcus** [comment]
@luizbills  kkkkkkkk

**Comment by caioincau** [comment]
Gosto de seguir a CGC
http://gcc.gnu.org/codingconventions.html#Spelling

"front end" para substantivo e "front-end" para adjetivo.

**Comment by felipefialho** [comment]
hahahaha.

**Comment by devvaldez** [comment]
A forma correta de escrever é Front-End.
Lembre-se que a palavra "Front-End" é um termo em inglês que se refere à parte da interface de um site ou aplicativo que o usuário vê e interage.
